github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
+============
+
+**Build Badges**
+
+Short project description. You can use the client's pitch. No more than ten lines.
+
+**Table of Contents**
+
+* [Setup](#setup)
+* [Development](#development)
+* [Deployment](#deployment)
+* [More Sections](#more-sections)
+* [Contribution Guidelines](#contribution-guidelines)
+* [Resources](#resources)
+* [Contacts](#contacts)
+
+## Setup
+
+A set of instructions to install the project. It can start with a list of dependencies, like databases, and finish with the instructions. For instance:
+
+First clone the repository to your file system:
+
+```
+git clone git@github.com:subvisual/guides.git
+```
+
+After, run the setup script:
+
+```
+bin/setup
+```
+
+To finish, open the file `.env` and replace `DB_USERNAME` and `DB_PASSWORD` with the correct username and password for the Postgres user.
+
+## Development
+
+A set of instructions to run the project in development. For instance:
+
+To start your development environment run:
+
+```
+bin/server
+```
+
+This will start the node application, Redis, and Gulp.
+ProjectName uses Gulp to watch and compile frontend assets.
+To build the assets only once, not watching for changes, run:
+
+```
+npm run build
+```
+
+## Deployment
+
+Deploys can be manual or automatic. In both cases, write a short explanation about how it works: if the process is manual write the steps necessary to accomplish it; if the process is automatic, link to the page to consult the deployment status.
+
+## More Sections
+
+This section is just a placeholder. You can replace it with any section or sections. Here you can write everything you feel the reader should know should about the project. This is the place to document other commands that are available to the user, or instructions on how to do something that's manual
+
+## Contribution Guidelines
+
+Contributions must follow [Subvisual's guides](https://github.com/subvisual/guides).
```

Comment:
I did use a `CONTRIBUTING.md` in a project before. They're handy because having them makes GitHub present a warning when opening a PR. Useful when we have contributors (clients or otherwise) that are not used to our way of doing things.